source/keymap.js

```diff
@@ -12,7 +12,8 @@
 			19  : 'pause',
 			412 : 'rewind',
 			417 : 'fastforward',
-			461 : 'back'
+			461 : 'back',
+			8	: 'back'	//webkit keycode of backspace key in keyboard
```

Comment:
In case of TV remote controller, there is no key which has key code 8.
As I commented, it means {{back space}} key in keyboard.
That's the everything that I can find from google, collab and some senior engineer from LSM.
How about e-mail to goldfinger-dev mailing list?